Assets/Placenote/Examples/ARKitHitTesting/MarkerManager.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.XR.iOS;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+
+namespace ARKitHitTesting
+{
+    /*========================================
+    * Classes to hold model information
+    ======================================== */
+
+    [System.Serializable]
+    public class ModelInfo
+    {
+        public float px; //position.x
+        public float py; //position.y 
+        public float pz; //position.z 
+    }
+
+    [System.Serializable]
+    public class ModelList
+    {
+        public ModelInfo[] models;
+    }
+
+
+    /*========================================
+     * Main Class for Managing Models
+    ======================================== */
+
+    public class MarkerManager : MonoBehaviour
+    {
+
+        public GameObject modelPrefab; // 3 prefabs are attached in the inspector
+
+        public List<ModelInfo> ModelInfoList = new List<ModelInfo>();
+        public List<GameObject> ModelObjList = new List<GameObject>();
+
+        /*====================================================
+        * Functions for adding and deleting models
+        ==================================================== */
```

Comment:
i can remove that. I had it in the shape manager because it helps me read the code better but it's not consistently everywhere so probably best to get rid of it

Reply:
btw these are not going into any documentation. Only files in LibPlacenote make their way into the api reference.

Reply:
done